src/public/modules/forms/admin/constants/update-form-types.js

```diff
@@ -0,0 +1,5 @@
+const UPDATE_FORM_TYPES = {
```

Comment:
this file can be in TS right? it'll save us some trouble later on

Reply:
True, but probably won't use this file in our react rewrite. Still will change to TS anyways

Reply:
8e20006